packages/expect-utils/src/jasmineUtils.ts

```diff
@@ -262,10 +262,7 @@ export function isImmutableUnorderedSet(maybeSet: any) {
 }
 
 export function isImmutableList(maybeList: any) {
-  return !!(
```

Comment:
This and following changes comes from Prettier. CI does not catch them, because of `/* eslint-disable */` in this file.

Reply:
ah, then we should make sure to include this file in https://github.com/facebook/jest/blob/8759d63787b832af1fc9ac0ec2cc57b6f325ba9a/package.json#L103

Reply:
Just check, I will send a separate PR removing that comment. Seems doable.

Reply:
we have purposefully not been running eslint on it to keep the diff from the jasmine equivalent down.

That might not be relevant 6-7 years later